setup.py

```diff
@@ -71,7 +71,16 @@
         **extra
     )
 except Exception as e:
-    if isinstance(e.message, str) and 'Could not run curl-config' in e.message:
+    if ((hasattr(e, 'args') and
```

Comment:
also just want to add that `isinstance(x, str)` won't work if x is a unicode object in Python 2, which may not matter for this particular exception anyways.